src/Tribe/Importer/Admin_Page.php

```diff
@@ -160,6 +160,16 @@ public function render_csv_tab() {
 					}
 				}
 				$import_type = get_option( 'tribe_events_import_type' );
+	
+				$import_type_titles_map = array();
+				
+				/**
+				 * Allows filtering the import type titles to go from a slug to a pretty title.
+				 * 
+				 * @param array $import_type_titles_map
+				 */
+				$import_type_titles_map = apply_filters('tribe_events_import_type_titles_map', $import_type_titles_map);
+				$import_type_title = isset($import_type_titles_map[$import_type]) ? $import_type_titles_map[$import_type] : ucwords( $import_type );
```

Comment:
Sorry to nitpick, but, spacing in the above two lines could do with a tweak (add spaces after opening brackets and before closing them etc) :grin: